test/integration/smoke/test_kubernetes_clusters.py

```diff
@@ -68,6 +68,9 @@
     "tier_netmask": "255.255.255.0"
 }
 
+K8S_VERSION_v1 = "1.25.0"
+K8S_VERSION_v2 = "1.26.0"
```

Comment:
should we get this from test_data?

Reply:
I'd like to keep this in the test script file.

Reply:
this has been moved to test_data.py as well